src/components/auth/SettingsModal.tsx

```diff
@@ -116,13 +121,50 @@ export const SettingsMenuItem = React.memo(function SettingsMenuItem({ finalRef
 
     const onSave = useCallback(() => {
         setFavoriteCommander(commanderSelectValue);
+
         closeModal();
     }, [closeModal, commanderSelectValue, setFavoriteCommander]);
 
     function moxfieldLinkerToggle() {
         setShowMoxfieldLinker(!showMoxfieldLinker);
     }
 
+    function updateMoxfieldInputValue(event: any) {
+        setMoxfieldInputValue(event.target.value);
+    }
+
+    const MoxfieldProfileFinder = MoxfieldService.useGetMoxfieldProfile();
+    const getMoxfieldProfile = async () => {
+        const moxfieldProfileObj = await MoxfieldProfileFinder(moxfieldInputValue);
+        console.log(moxfieldProfileObj);
```

Comment:
nit: remove console log

Reply:
Leaving for now - will remove after step 3